tests/data/bundles/test_core.py

```diff
@@ -303,7 +303,7 @@ def _(environ,
                   output_dir):
                 _wrote_to.append(output_dir)
 
-        _wrote_to.clear()
+        _wrote_to[:] = []
```

Comment:
What was this change for?  Also, is it intentional that the default value for `_wrote_to` is mutable?